### PR TITLE
DOM steps

### PR DESCRIPTION
**Pull request by lolmaus**

### CODE REVIEW

File: addons/steps-dom/tests/integration/steps-test.ts [hunk 1]
```diff
@@ -80,8 +78,274 @@ module('Integration | Component | steps', function (hooks) {
...
   });
+
+  //
+
+  module('When I move the mouse pointer into $emberBddLabel', function () {
+    interface TestContextMouseEnter extends TestContext {
+      action?: SinonSpy;
+    }
+
+    test('Normal', async function (this: TestContextMouseEnter, assert) {
```
Comment (lolmaus) [inline addons/steps-dom/tests/integration/steps-test.ts:274]: @simonihmig This test in `addons/steps-dom` fails for the mouseenter step, can't figure out why.

Can you please have a look when you have time?

Comment (lolmaus) [reply addons/steps-dom/tests/integration/steps-test.ts:274]: The specific issue is that the action `{{on "mouseenter" this.action}}` won't fire.

Comment (simonihmig) [reply addons/steps-dom/tests/integration/steps-test.ts:274]: @lolmaus I don't see a reason why this shouldn't work. `mouseenter` is a special event in that it doesn't bubble up. That was an issue when using the event delegation pattern, i.e. registering a single event listener at a root element (e.g body) to catch all events below it. A while ago I did a PR to enable mouseenter events in Ember's EventDispatcher without jQuery, which was not working for this reason. The test coverage was involving, as I had to replicate what the events fired by a real browser would be, see https://github.com/emberjs/ember.js/pull/16999/files#diff-01b9a4fcff882974a0d09c7e5bd8a38a1729802457d197a8746b0b51374fb0b8. But I don't know if this is of any relevance here. With `{{on ...}}` we don't rely on the EventDispatcher and thus event delegation, so this shouldn't be an issue...

What puzzles me: CI is actually green! So does this test not run in CI yet? Or does it only fail locally in a real browser?

Comment (lolmaus) [reply addons/steps-dom/tests/integration/steps-test.ts:274]: This was a typo in `mouseeenter`. 😔

**Pull request merged**